StudCity.Frontend/src/pages/Authenticate/index.jsx

```diff
@@ -29,8 +29,8 @@ function Authenticate() {
 	return (
 		<div className="w-1/1 h-screen flex">
 			<div className="w-1/2 h-screen flex bg-primatyWhite">
-				<div className="container flex flex-col w-4/5 m-auto h-[85%] bg-primatyWhite rounded-3xl p-11 shadow-md ">
-					<h1 className="w-full text-center font-bold text-4xl my-3 text-primaryAuthentication">Welcome back</h1>
+				<div className="container flex flex-col w-2/3 m-auto h-[80%] justify-between bg-primatyWhite rounded-3xl p-11 shadow-md ">
```

Comment:
Maybe we can use h-4/5 ? @Serhiy-dotcom @Vova-Romaniuk

Reply:
Yes great suggestion, docs says we definitely can